README.md

```diff
@@ -25,4 +25,31 @@ go build -o sst cmd/main.go
 ./sst [sample-dir]
 ```
 
-Make sure to authorize the gcloud SDK and set a default project before running this program.
+Make sure to authorize the gcloud SDK and set a default project and Cloud Run region before running this program. A
+default Cloud Run region can be set by setting the `run/region` gcloud property.
+
+### README parsing
+If you'd like, make sure to include the following comment code tag immediately preceding code blocks to customize how
+the program should build and deploy your sample:
+
+```text
+[//]: # ({sst-run-linuxmacos})
+```
+
+For example:
+````text
+[//]: # ({sst-run-linuxmacos})
+```
+gcloud builds submit --tag=gcr.io/${GOOGLE_CLOUD_PROJECT}/run-mysql
+```
+````
+
+Do not set the Cloud Run region you'd like to deploy to through the `--region` flag in the `gcloud run` commands.
+Instead, as mentioned above, do so by setting the `run/region` gcloud property.
+
+In addition to setting a default Cloud Run region, make sure to deploy to the fully platform on Cloud Run. You can
```

Comment:
' ... fully platform ' 

should be

' ... fully managed platform '

Reply:
Resolved, thank you.